src/app/portlets/dot-edit-content/dot-edit-content.component.ts

```diff
@@ -41,26 +45,24 @@ export class DotEditContentComponent implements OnInit {
         private dotGlobalMessageService: DotGlobalMessageService
     ) {}
 
-
-
     ngOnInit() {
         this.dotLoadingIndicatorService.show();
 
         this.route.data.pluck('editPageHTML').subscribe((editPageHTML: DotRenderedPage) => {
             this.pageIdentifier = editPageHTML.identifier;
             this.dotEditContentHtmlService.initEditMode(editPageHTML.render, this.iframe);
 
-            this.dotEditContentHtmlService.contentletEvents.subscribe((res) => {
+            this.dotEditContentHtmlService.contentletEvents.subscribe((contentletEvent: any) => {
```

Comment:
we should create a interface for 'contentletEvent'

Reply:
Totally, we do that in the refactor, specially because the way we are doing events from the iframe will change.